src/lib/checkbox/checkbox.scss

```diff
@@ -1,14 +1,149 @@
-@use '../theme';
-@use './checkbox.mixins' as mixins;
-@use '../form-field/mixins' as form-field-mixins;
-
-@include mixins.core-styles;
-@include form-field-mixins.shadow-styles;
+@use './core';
+@use './configuration';
+@use '../core/styles/tokens/checkbox/tokens';
```

Comment:
Let's remove this tokens import so it doesn't give the impression we should be using tokens directly here